groups/groups.yaml

```diff
@@ -1513,3 +1513,14 @@ groups:
       - natashasarkar@google.com
       - extaeger@google.com
       - samwronski@google.com
+
+  - email-id: k8s-infra-staging-iscsi@kubernetes.io
+    name: k8s-infra-staging-iscsi
```

Comment:
We actually have k8s-infra-staging-sig-storage where we've been publishing all our sig projects under. I think we can use that instead.

Reply:
@huffmanca seems like it just needs the changeto infra-staging as mentioned about and then should be good to go?

Reply:
I added the iscsiplugin to the `k8s.gcr.io/images/k8s-staging-sig-storage/generate.sh` script. Checking the other repos here, I didn't see specific information for them under `k8s.gcr.io/images` or `k8s.gcr.io/manifests`, so I've removed these references.